integration_tests/e2e/overview/goal/createGoals.cy.ts

```diff
@@ -237,15 +237,15 @@ context('Create goals', () => {
         .withRequestBody(
           matchingJsonPath(
             '$[?(@.goals.size() == 2 && ' +
-              "@.goals[0].prisonId == 'MDI' && " +
+              "@.goals[0].prisonId == 'BXI' && " +
```

Comment:
Because the stub prisoners have been changed to be in BXI (see change below), I've had to change these tests to reflect that 👍